cxx_mkspecs/gxx_mkspecs.py

```diff
@@ -238,6 +238,36 @@ def cxx_raspberry_gxx49_arm(conf):
 
 
 @conf
+def cxx_raspberry2_gxx49_arm(conf):
+    """
+    Detect and setup the g++ cross-compiler for Raspberry Pi 2 (Linux)
+    """
+    conf.env['CXX'] = 'arm-linux-gnueabihf-g++-4.9'
+    conf.mkspec_gxx_configure(4, 9)
+    # atomic support that is required for std::threads (without this flag,
+    # the threading code might call pure virtual methods)
+    conf.env['LINKFLAGS'] += ['-march=armv7-a']
+    # Note: libstdc++ might not be available on the target platform
+    # Statically link with the C++ standard library
+    conf.env['LINKFLAGS'] += ['-static-libstdc++']
+
+
+@conf
+def cxx_raspberry2_gxx_arm(conf):
```

Comment:
Btw, I would be against adding such a "default" mkspec for a platform like this.
We always specify the version numbers for a given toolchain, so this would go against that pattern.